statsmodels/tsa/tests/test_holtwinters.py

```diff
@@ -358,6 +378,56 @@ def test_hw_seasonal_buggy(self):
                             2)
 
 
+    def test_simulate(self):
+        """
+        Test for :meth:``statsmodels.tsa.holtwinters.HoltWintersResults``.
+
+        The tests are using the implementation in the R package ``forecast`` as
+        reference, and example data is taken from ``fpp2`` (package and book).
+
+        Each method is tested, the methods are prepended with a comment
+        describing the R code that was used to obtain the reference results.
+
+        Each test should also set the seed back to 0, so the same ``innov``
+        values can be used in R.
+        """
+
+        # TODO:
+        # - test input parameters
+        # - test all stable models
+
+
+        # ETS(M, A, M) with parameters obtained from R:
+        #
+        # fit <- ets(austourists, model = "MAM")
+        # innov <- c(1.76405235, 0.40015721, 0.97873798, 2.2408932)
+        # sim <- simulate(fit, innov = innov*fit$sigma)
+        expected = [78.06170, 49.01698, 61.71652, 67.60118]
+
+        fit = ExponentialSmoothing(
+            self.austourists, seasonal_periods=4, trend="add", seasonal="mul",
+            damped=False
+        ).fit()
+
+        np.random.seed(0)
+        nsim = 100
+        steps = 20
+        sim = fit.simulate(steps, nsim=nsim, error="mul")
+
+        import matplotlib.pyplot as plt
```

Comment:
Make a plotting only test.

Reply:
Sorry, I should have removed this before. This was just for myself as a first visual sanity check, not a real test. I removed it now.

Reply:
Sorry, I forgot to remove this, this was not thought of as a real test, but a first visual sanity check for me. I removed it now.